packages/manager/modules/billing-components/src/components/commitment/commitment.constants.js

```diff
@@ -1,6 +1,7 @@
 export const CATALOG_MAPPING = {
   '/dedicated/server/{serviceName}': 'baremetalServers',
   '/vps/{serviceName}': 'vps',
+  '/license/cpanel/{serviceName}': 'licensecPanel',
```

Comment:
Catalog doesn't look prodded yet, maybe issue is commitment are brought up as they shouldn't

Reply:
It will be put in production. I have created a request as per their request https://projects.dsi.ovh/servicedesk/customer/portal/7/DTRSD-44385